src/pages/projects/an-archival-experience-celebrating-c-p-companys-50th-anniversary.md

```diff
@@ -0,0 +1,181 @@
+---
+templateKey: Project
+title: An Archival Experience celebrating <br>C.P. Company’s 50th Anniversary
+intro: >-
+  We were proud to join C.P. Company’s 50th anniversary Cinquanta celebrations to help them bring their rich archive to life – in both a physical and digital form. Working with five-decade’s worth of their hybrid urban sportswear, objects, sketches and editorials, we created an interactive spatial archive in a repurposed car park in Darwen, UK, as part of the 2021 British Textile Biennale, as well as a dynamic online archive. A hybrid of physical and digital space, the exhibition offers visitors an embodied experience of the often-hidden brand archive.
+content:
+  - video:
+      autoplay: false
+      hasControls: false
+      loops: true
+      isMuted: true
+      isAlwaysMuted: false
+    marginTop: 0
+    marginLeft: 0
+    zIndex: 0
+    image: /img/top-grid@2x.png
+    width: 100
+  - caption: |-
+      <div class="content-section-title">‘Cinquanta’ Archive Exhibition</div>
```

Comment:
We needed special styling (larger titles in a text block) for this specific project. I've added the styles in a special section of global.css